### DISCUSSION TITLE
Feature Request: Import & Export  / Sync configurations

### DISCUSSION THREAD

**Discussion by obarlev** [notaapp/talk#220] [Feature Requests]
I've made many changes in the shortcuts section so Nota will work in a similar way to my code Editors (Sublime & PyCharm).
Per my check, there is no way to sync / transfer those changes to my other machines
I would be great if you provide such capability, even a simple settings file import & export a good option.

**Comment by astoilkov** [comment]
This is something we are already looking into. We have two ideas:
- Automatic syncing of configurations by linking it with your license. The user enters the license and the configurations are automatically synced.
- Manual Export/Import. You run the Export command from the first computer and then on the second computer you run the Import command.

Which one do you like more?

**Comment by borekb** [comment]
I'm in a similar situation, configuring Nota on multiple computers is a bit annoying.

A couple of ideas:

- Workspace settings in a file like `<myworkspace>/.nota/settings.json` would be good enough for me as I mostly just use a single workspace (`~/Google Drive/Notes`).

- [Settings Sync](https://marketplace.visualstudio.com/items?itemName=Shan.code-settings-sync) was a popular way to sync VSCode settings before the functionality was built-in; it used Gists which I found both convenient and quite powerful, for example, I could see how my settings evolved over time.

- I also use "dotfiles" to configure various programs, if Nota had a predictable location like `~/.nota-settings.json` I would utilize my `dotfiles` repo to deal with that.

Automatic sync tied to a license of course sounds logical and convenient, just wanted to mention a couple of other options 😄. Export/import isn't ideal for me personally (too much work 😄).

**Comment by astoilkov** [comment]
We are also considering your third option. A `.nota` folder in the root `~`. This will also enable us to have global extensions available regardless of the current workspace.

**Reply by Arbiter313** [reply]
If you do decide to do this I would be eternally grateful if you put the folder in`~/.config` instead 🙏😅
So much cleaner!

**Reply by panckreous** [reply]
+1 for `~/.config` for anyone counting.
